components/proxy/conf/Caddyfile

```diff
@@ -146,6 +146,13 @@
 	}
 }
 
+# Internal configcat endpoint
+:9547 {
```

Comment:
Do we need a seperate port? could it be part of the 9546 port above?

Reply:
I thought about it but then service port has already name `analytics`. Should we rename it? Is it fine?

Reply:
It also has slightly different network policies. So not sure.